pycheribuild/projects/cross/qt5.py

```diff
@@ -433,6 +433,7 @@ def __init__(self, config):
             # self.configure_args.append("--disable-extras")  # can't add this to host build, it will fail otherwise
             # We have modified the ICU data Makefile so that ICU builds a big endian data archive
             self.configure_args.append("--with-data-packaging=archive")
+            self.COMMON_FLAGS.append("-fPIC")  # needed for morello hybrid webkit not to complain about relocations
```

Comment:
Probably wants to be -fPIE not -fPIC, no?

Reply:
(although they should fix their toolchain to support PDEs properly... and an XXX: and maybe only do this for aarch64?)

Reply:
I'm not sure if the build system produces any, but the flag also applies to shared libraries, so probably -fPIC is safer.